src/engine/ruleset/integrations/windows/decoders/windows-security.yml

```diff
@@ -27,37 +27,39 @@ normalize:
       - wazuh.decoders: array_append(windows-security)
```

Comment:
Please update the metadata compatibility.

Reply:
Done